### ISSUE TITLE
Upgrade to Pixi v7

### ISSUE THREAD

**Issue by Bonkles** [facebookincubator/RapiD#617]
Using the migration guide, we need to update things to use Pixi v.7 (currently in beta). 

Resources: 

https://github.com/pixijs/pixijs/wiki/v7-Migration-Guide

Also, their site docs call out the fact that `@pixi` and any other deps need to be uninstalled and reinstalled - you can NOT just update the package.json and hope for the best. 

**Update: This work is going into a branch called 'pixi-v7'.**

**Comment by Bonkles** [comment]
Looks like there are problems with the v7 beta 3 of pixi, possibly, or there are problems with the @pixi-essentials/texture-allocator mangling meshes and invalidating/nullifying the texture's mesh/shader property. 

Current behavior- things draw fine upon first render, but as soon as something needs to be destroyed or recalculated (say, a map pin goes off screen and comes back) the BitmapText `updateText()` method will explode when it tries to retrieve the texture from the BitmapText's own cache. 


Call stack of the offending issue: 

```
set texture (/Users/clarkben/working/RapiD/node_modules/@pixi/mesh/src/Mesh.ts:246)
updateText (/Users/clarkben/working/RapiD/node_modules/@pixi/text-bitmap/src/BitmapText.ts:424)
validate (/Users/clarkben/working/RapiD/node_modules/@pixi/text-bitmap/src/BitmapText.ts:652)
getLocalBounds (/Users/clarkben/working/RapiD/node_modules/@pixi/text-bitmap/src/BitmapText.ts:639)
placePointLabel (/Users/clarkben/working/RapiD/modules/pixi/PixiLayerLabels.js:450)
<anonymous> (/Users/clarkben/working/RapiD/modules/pixi/PixiLayerLabels.js:383)
placePointLabels (/Users/clarkben/working/RapiD/modules/pixi/PixiLayerLabels.js:364)
render (/Users/clarkben/working/RapiD/modules/pixi/PixiLayerLabels.js:216)
render (/Users/clarkben/working/RapiD/modules/pixi/PixiScene.js:131)
_app (/Users/clarkben/working/RapiD/modules/pixi/PixiRenderer.js:419)
_tick (/Users/clarkben/working/RapiD/modules/pixi/PixiRenderer.js:273)
emit (/Users/clarkben/working/RapiD/node_modules/@pixi/ticker/src/TickerListener.ts:67)
update (/Users/clarkben/working/RapiD/node_modules/@pixi/ticker/src/Ticker.ts:422)
Ticker._tick (/Users/clarkben/working/RapiD/node_modules/@pixi/ticker/src/Ticker.ts:124)
requestAnimationFrame (Unknown Source:0)
Ticker._tick (/Users/clarkben/working/RapiD/node_modules/@pixi/ticker/src/Ticker.ts:128)
requestAnimationFrame (Unknown Source:0)
Ticker._tick (/Users/clarkben/working/RapiD/node_modules/@pixi/ticker/src/Ticker.ts:128)
requestAnimationFrame (Unknown Source:0)
Ticker._tick (/Users/clarkben/working/RapiD/node_modules/@pixi/ticker/src/Ticker.ts:128)
```

**Comment by Bonkles** [comment]
Tried to create a codesandbox to repro the problem outside of RapiD: https://codesandbox.io/s/pixi-playground-forked-mrkxxq?file=/src/index.js

This sandbox depends on pixi.js 7 beta 3 and I am able to create a bitmap font, dirty it by changing the letter spacing, and re-render on each ticker update. no issues.

**Comment by Bonkles** [comment]
Created an issue on the pixi.js site and it looks like they already fixed it! 

https://github.com/pixijs/pixijs/issues/8759

In the meantime, I have worked around this bug by setting `children:false` in each call to `destroy` for our BitmapText labels.

**Comment by Bonkles** [comment]
Fixed in 8b3761b70770161a711dc9f00425888690507b74.
